package.json

```diff
@@ -25,6 +25,8 @@
     "@kiwicom/margarita-relay": "^0",
     "better-npm-run": "^0.1.1",
     "expo": "^31.0.6",
+    "graphql-iso-date": "^3.6.1",
+    "luxon": "^1.8.2",
```

Comment:
Why is this dependency moved here?

Reply:
We use luxon in the `core` and also in the `graphql` directories. `graphql-iso-date` shouldn't be here, because it's only in `graphql`, so I will change it.

Reply:
I suppose that this is the right place for `luxon`, because we want to work with the same version in all workspaces.

Reply:
And that means it would be easy to replace it here with `date-fns` 😉

Reply:
OK